internal/gke/cluster_info.go

```diff
@@ -79,15 +79,15 @@ type ClusterInfo struct {
 // GetClusterInfo returns information about a GKE cluster running in a given
 // cloud region.
 // REQUIRES: The caller is running on the user machine.
-func GetClusterInfo(ctx context.Context, config CloudConfig, cluster, region string) (*ClusterInfo, error) {
+func GetClusterInfo(ctx context.Context, config CloudConfig, region string) (*ClusterInfo, error) {
 	// Fetch cluster credentials to the local machine.
 	kubeFileName := filepath.Join(
-		os.TempDir(), fmt.Sprintf("serviceweaver_%s_%s", cluster, uuid.New().String()))
+		os.TempDir(), fmt.Sprintf("serviceweaver_%s_%s", applicationClusterName, uuid.New().String()))
```

Comment:
`applicationClusterName == "serviceweaver"`

Maybe we just drop this parameter? No need to create a file `serviceweaver_serviceweaver_1234`.

Reply:
Done